src/Lachain.Benchmark/BlockchainBenchmark.cs

```diff
@@ -181,7 +206,7 @@ private static void _Benchmark(string text, Func<int, int> action, uint tries)
                 var blockWithTxs = blocks[i];
                 // blockchainManager.PersistBlockManually(blockWithTxs.Block, blockWithTxs.Transactions);
                 return i;
-            }, (uint) blocks.Length);
+            }, (uint)blocks.Length);
 
             var elapsedTime = TimeUtils.CurrentTimeMillis() - currentTime;
```

Comment:
Print this time too

Reply:
this is the old benchmark function, and not touched; any ways the time is already printed